docs/funding/google-season-of-docs-2023.md

```diff
@@ -0,0 +1,134 @@
+
+## Your project’s scope
+
+<!---
+Tell us about what documentation your organization will create, update, or improve. If some work is deliberately not being done, include that information as well. Include a time estimate, and whether you have already identified organization volunteers and a technical writer to work with your project.
+-->
+
+This project includes:
+
+- Creation of an accessibility section in JupyterLab documentation containing default accessibility settings and customizations inspired by https://code.visualstudio.com/docs/editor/accessibility
+- Create user-focused documentation that summarizes the state of JupyterLab areas that have been audited for accessibility.  This includes an approachable report on keyboard navigation and 400% zoom audits results.
+
+As a stretch goal, we would like to include:
+
+- An authoring guide focused on best practices to generate accessible notebooks.
+
+Work that is out of the scope of this project includes:
+
+- Creation of developer-focused documentation
+- Tutorials and workshops to present in conferences
+- Live streams and social media engagement
+
+The developer that will be mentoring this project has previously mentored successful GSoD in another project last year and has been involved in the accessibility remediation work for JupyterLab. We don’t have a technical writer for this project, but we intend to start the  application and selection process once we receive the acceptance by Google Season of Docs.
+
+We estimate that this project will be completed by one technical writer working full-time in a span of six months.
+
+## Measuring your project’s success
+
+<!---
+How will you know that your new documentation has helped solve your problem? What metrics will you use, and how will you track them?
+-->
+
+Even though JupyterLab users vary in their experience, knowledge and confidence of the application, there’s an overall unfamiliarity about the accessibility features available within it. This is mostly a result of the lack of a reliable and official documentation that can guide users through different situations like adjusting theming, keyboard support and assistive technologies support. There have been numerous questions raised about accessibility features in the issue tracker of the project and in general, the community wants to move to a more inclusive access to the information given that JupyterLab is one of the most common tools used in education, research and scientific development.
+
+We will track three metrics to measure this documentation project's success. The first is the number of accessibility features documented; the second is the number of pages added regarding the audits for keyboard navigation and 400% zoom.
```

Comment:
This mentions three metrics - though I can only see two

1. No features documented
2. Zoom and keyboard pages

Seems this needs adjusting

```suggestion
We will track two metrics to measure this documentation project's success. The first is the number of accessibility features documented; the second is the number of pages added regarding the audits for keyboard navigation and 400% zoom.
```